src/examples/ParkingLot/Disclaimer.txt

```diff
@@ -0,0 +1,33 @@
+% -------------Software Disclaimer---------------
```

Comment:
same as above, there is already a LICENSE file

Reply:
please dont remove this.